pages/nfttest/fetch/index.vue

```diff
@@ -307,6 +322,7 @@ export default class FetchIndex extends Vue {
           this.localeLocation({
             name: 'nfttest-mint-iscnId',
             params: { iscnId: this.iscnId },
+            query: this.mintQuery,
```

Comment:
Any reason why not use `params` instead of `query`?

Reply:
Currently, it's `/mint/{iscnId}?ogImageUrl={ogImageUrl}`. Should we make it like `/mint/{iscnId}/{ogImageUrl}`? I think `ogImageUrl` is a passing value for `iscnId`. And we don't want to change the file structure just for accepting `ogImageUrl`?

Reply:
The trick is you dont need a param to be in the path, but you can still pass it as a param as long as it is defined in the page, and passed when the page is called in router

Reply:
i.e. try just putting `params: { iscnId: this.iscnId, ogImageUrl: this.crawledData.image }` and see if `$route.params.ogImageUrl` exists in the next page

Reply:
I see, it works! Thanks!!